packages/meteor-accounts-saml/saml_server.js

```diff
@@ -182,61 +186,64 @@ middleware = function (req, res, next) {
 			return;
 		}
 
-		if (!samlObject.actionName)
-			throw new Error("Missing SAML action");
+		if (!samlObject.actionName) {
+			throw new Error('Missing SAML action');
+		}
 
-		console.log(Accounts.saml.settings.providers)
-		console.log(samlObject.serviceName)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Accounts" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313313354/issues/source?bid=3199634&fileBranchId=3199636#l193)